### PR TITLE
Add test for heterogeneous conductivity

### PR DESCRIPTION
**Pull request by MasanoriKanamaru**
Added a test of a case where thermal conductivity is heterogeneous over the surface of the asteroid.

### CODE REVIEW

File: test/thermal_property_distribution.jl [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@testset "thermal_property_distribution" begin
```
Comment (hyrodium) [inline test/thermal_property_distribution.jl:1]:
```suggestion
The following tests are almost the same as `TPM_Ryugu.jl`. The only difference is thermophysical property distribution.
@testset "thermal_property_distribution" begin
```

File: test/thermal_property_distribution.jl [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    ##= TPM =##
+    # When thermophysical properties vary from facet to facet
+    # Northern hemisphere:
+    #     A_B = 0.04
+    #     k   = 0.1
+    #     ε   = 1.0
+    # Southern hemisphere:
+    #     A_B = 0.1
+    #     k   = 0.3
+    #     ε   = 0.9
```
Comment (hyrodium) [inline test/thermal_property_distribution.jl:69]:
```suggestion
    # "Northern" hemisphere:
    #     A_B = 0.04
    #     k   = 0.1
    #     ε   = 1.0
    # "Southern" hemisphere:
    #     A_B = 0.1
    #     k   = 0.3
    #     ε   = 0.9
```
* These wordings such as "Northern" are expedient usage, so quoting them would be better.
* Could you specify the units and meanings of `A_B`, `k`, and `ε`? (e.g. `ε = 0.9[W/m²] (emissivity)`)

**Review by hyrodium — COMMENTED** [review]
I added a minor comments on the change.

File: test/thermal_property_distribution.jl (hunk 1 above)
Comment (MasanoriKanamaru) [reply test/thermal_property_distribution.jl:2]: I added the comment at the top of the test file.

File: test/thermal_property_distribution.jl (hunk 2 above)
Comment (MasanoriKanamaru) [reply test/thermal_property_distribution.jl:70]: I corrected the comment and added the units, except for unitless albedo and emissivity.

**Review by hyrodium — APPROVED** [review]
LGTM!

**Pull request merged**